scripts/nolints.json

```diff
@@ -1037,7 +1037,7 @@
  ["docBlame", "StrictWeakOrder.Equiv"],
  ["docBlame", "StrictWeakOrder.«term_≈[_]_»"],
  ["docBlame", "Structomorph.mem_groupoid"],
- ["docBlame", "StructureGroupoid.eq_on_source'"],
+ ["docBlame", "StructureGroupoid.mem_of_eqOnSource'"],
```

Comment:
Can you add a docstring to `mem_of_eqOnSource'` instead? Now that you've thought about this, it shouldn't be too hard, right?

Reply:
Both the added lemma and docstrings are now on #9827.